parcel_test.go

```diff
@@ -28,60 +24,89 @@ func getTestParcel() Parcel {
 	}
 }
 
-// TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer db.Close()
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	parcel.Number, err = store.Add(parcel)
+
+	require.NoError(t, err)
+	require.NotEmpty(t, parcel.Number)
+
+	stored, err := store.Get(parcel.Number)
+	require.NoError(t, err)
+
+	assert.Equal(t, parcel, stored)
 
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	err = store.Delete(parcel.Number)
+	require.NoError(t, err)
 
-	// delete
-	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+	_, err = store.Get(parcel.Number)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для проверки ошибок нужно использовать специализированные функции
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```

Reply:
нужно было старый вариант убрать